static/src/javascripts/projects/common/modules/commercial/creatives/add-tracking-pixel.js

```diff
@@ -0,0 +1,17 @@
+define([
+    'common/utils/template',
+    'text!common/views/commercial/tracking-pixel.html'
+], function (
+    template,
+    trackingPixelStr
+) {
+    var trackingPixelTpl;
+    function addTrackingPixel($adSlot, url) {
+        if (!trackingPixelTpl) {
```

Comment:
I keep coming back to it, although I also used `trackingPixelTpl || (trackingPixelTpl = template(trackingPixelStr));` which I find more efficient. But thanks, doing it!